04LessonFour/in-class-project/index.html

```diff
@@ -2,72 +2,89 @@
 
 <html>
 
-<head>
+  <head>
 
-    <title>The Daily Planet</title>
-    <link rel="stylesheet" href="css/style.css">
+    <title>Intel &reg;</title>
 
-</head>
+    <link rel="stylesheet" href="css/style.css">
 
-<body>
+  </head>
 
+  <body>
+    <!--Still curious about fonts. These do not match the previous page, but replicating it was not as simple as imagined. Change later when more fonts are installed.  -->
     <header>
 
```

Comment:
try one of the generic font families for now. we'll worry about other fonts later.